js/gmcqModel.js

```diff
@@ -0,0 +1,3 @@
+import McqModel from 'components/adapt-contrib-mcq/js/mcqModel';
```

Comment:
doing this in a separate file seems slightly OTT to me... but if you were asked to do it this way then feel free to ignore me !

Reply:
Yeah, Ollie and I had a chat about this and decided on a separate file to emphasise structure (I also like that all components will have the same number of js files now to make separation clearer).